unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ProfileHUD/IProfileHUDView.cs

```diff
@@ -0,0 +1,41 @@
+
+using System;
+using UnityEngine;
+
+public interface IProfileHUDView
+{
+    event EventHandler SignedUpPressed;
+    event EventHandler LogedOutPressed;
+    event EventHandler ClaimNamePressed;
+
+    event EventHandler Opened;
+    event EventHandler Closed;
+    event EventHandler<string> NameSubmitted;
+    event EventHandler<string> DescriptionSubmitted;
+
+    event EventHandler ManaInfoPressed;
+    event EventHandler ManaPurchasePressed;
+
+    event EventHandler TermsAndServicesPressed;
+    event EventHandler PrivacyPolicyPressed;
+
+    public GameObject GameObject { get; }
+    public RectTransform ExpandedMenu { get; }
+    public RectTransform TutorialReference { get; }
```

Comment:
Because we are in an interface I think that the public is redundant

Reply:
Agree and changed